prowler/lib/check/models.py

```diff
@@ -38,6 +38,7 @@ class Check_Metadata_Model(BaseModel):
     CheckID: str
     CheckTitle: str
     CheckType: list[str]
+    CheckAlias: list[str] = []
```

Comment:
- Could you leave it as an `Optional` instead of a empty list?
- Is this field included in the outputs?